agents/mysql/perfschema/perfschema.go

```diff
@@ -301,11 +301,12 @@ func makeBuckets(current, prev map[string]*eventsStatementsSummaryByDigest, l *l
 		}
 
 		count := inc(currentESS.CountStar, prevESS.CountStar)
+		fingerprint, _ := truncate.Query(*currentESS.DigestText)
```

Comment:
Do we need check isTruncated in these cases where you using truncate.Query?

Reply:
Yes, good point - we can at least pass it to QAN for future use.